mirage/config.js

```diff
@@ -27,4 +22,173 @@ export default function () {
     return schema.challenges.where({ assessmentId: request.params.id });
   });
   this.get('/challenges/:id');
+
+  this.passthrough('https://api.airtable.com/**');
+
+}
+
+export function testConfig() {
+  this.get('https://api.airtable.com/v0/appHAIFk9u1qqglhX/Tests', function () {
+    return {
+      "records": [
```

Comment:
move this to a fixture: [ref fixtures](http://www.ember-cli-mirage.com/docs/v0.2.x/seeding-your-database/#fixtures) + [ref loadFixtures](http://www.ember-cli-mirage.com/docs/v0.2.x/configuration/#loadFixtures)